README.md

```diff
@@ -71,7 +71,7 @@ The configuration is made by using k8s controller annotations under the root tem
 | pod-watcher | pod-kubeobserver.io/ignore | boolean | pod watcher will ignore all the pod events | false |
 | pod-watcher | pod-init-container-kubeobserver.io/watch | boolean | pod watcher will trigger events for init containers related to the pod | false |
 | *All* | kubeobserver.io/receivers | comma separated string | a comma separated string of recevier names that the events will be publish to. unknown names will be ignored | default recevier is defined in kubeobserver using DEFAULT_RECEIVER env variable |
-| pod-watcher | pod-update-kubeobserver.io/ignore | boolean | pod watcher will ignore 'Update' events and notify only on 'Add'/'Delete' events | false |
+| pod-watcher | pod-update-kubeobserver.io/watch | boolean | pod watcher will ignore 'Update' events and notify only on 'Add'/'Delete' events | false |
```

Comment:
i don't think it's a good description. the description should describe what this feature does when is active.
so if use change the suffix to watch then the describe should be something like this:
`pod watcher will report on pod 'Update' events` (and the default is false)

Reply:
forgot about the description